spec/models/show_spec.rb

```diff
@@ -1,5 +1,29 @@
+# frozen_string_literal: true
+
 require 'rails_helper'
 
 RSpec.describe Show, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  let(:show) { create(:show) }
+  let(:title) { show.title }
+  let(:show_type) { show.show_type }
+
+  it "is a valid with the presence of show_type, show_id and title" do
+    expect(show).to be_valid
```

Comment:
this is already after creation, you should be doing something like this:
`expect(build(:show, title: nil)).to_not be_valid`

Reply:
or even better just use https://github.com/thoughtbot/shoulda-matchers for testing simple validations and doing something like:
`it { should validate_presence_of(:name) }`

Reply:
Oh that gem is nice!! Thank you!